azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py

```diff
@@ -65,18 +66,70 @@ class ColumnMapping:
     pre-created (it is recommended to create the mappings in advance and use ingestionMappingReference).
     To read more about mappings look here: https://docs.microsoft.com/en-us/azure/kusto/management/mappings"""
 
-    # Json Mapping consts
     PATH = "Path"
     TRANSFORMATION_METHOD = "Transform"
-    # csv Mapping consts
     ORDINAL = "Ordinal"
     CONST_VALUE = "ConstValue"
-    # Avro Mapping consts
     FIELD_NAME = "Field"
+
+    # TODO - these are undocumented - do they really exist?
```

Comment:
Columns is deprecated
StorageDataType seems like internal usage - delete both

Reply:
Done